common/views/components/Buttons/Button/Button.js

```diff
@@ -37,6 +40,17 @@ const Button = ({
   const fontClasses = extraClasses && extraClasses.indexOf('btn--tertiary') > -1
     ? {s: 'HNM5'}
     : {s: 'HNM4'};
+  function handleClick(e) {
+    if (clickHandler) {
+      clickHandler(e);
+    }
+    if (trackingEvent) {
+      trackEvent(trackingEvent);
+    }
+    if (trackingEventV2) {
+      trackEventV2(trackingEventV2);
+    }
+  }
```

Comment:
Hmmm. What about putting component tracking (`category: Button`) in here by default, and add the download specific tracking in the `clickHandler`?